src/DestroyCommand.php

```diff
@@ -21,9 +21,8 @@ protected function configure()
     /**
      * Execute the command.
      *
-     * @param  \Symfony\Component\Console\Input\InputInterface  $input
-     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
-     * @return void
```

Comment:
@GrahamCampbell I used .php_cs from laravel/framework and it's still removed all the ``` @return void``` lines. Ideas?